conf/flistd.yaml

```diff
@@ -0,0 +1,3 @@
+exec: flistd -broker unix://var/run/redis.sock
```

Comment:
Let stays consistent. Or we specify it for all modules daemon or for none.
Also, since some of the module requires to have some cache fs. We should maybe also set the `-root` flag of some modules daemon

Reply:
yeah, i agree i already provided it to all modules.